sonar-erlang-plugin/src/main/java/org/sonar/plugins/erlang/settings/ErlangLanguageProperties.java

```diff
@@ -35,7 +36,7 @@ private ErlangLanguageProperties() {
   }
 
   public static List<PropertyDefinition> getProperties() {
-    return asList(PropertyDefinition.builder(FILE_SUFFIXES_KEY)
+    return Collections.singletonList(PropertyDefinition.builder(FILE_SUFFIXES_KEY)
```

Comment:
What's the benefit to change this to such a type?

Reply:
In this case it is to make it more clear what this static method actually returns.
`asList` is too broad as it is a very general "bridge" between fixed sized arrays and lists, but `singletonList` makes it very clear that we return a list with only one element of type `PropertyDefinition`.